### ISSUE TITLE
NetCDF-C v4.9.0 Release

### ISSUE THREAD

**Issue by WardF** [Unidata/netcdf-c#2344]
Playing with 'new' GitHub tools, workflows for managing the `v4.9.0` release.  New `Project` here: https://github.com/orgs/Unidata/projects/4

**Comment by Alexander-Barth** [comment]
I am wondering if it would be possible to make a release candidate first (when ready)?

**Comment by Alexander-Barth** [comment]
OK, I understand. Thank you for your feed-back.

**Comment by DennisHeimbigner** [comment]
Also, we used to do release candidates, but never got any feedback. Apparently
no one ever looked at them.

**Comment by gsjaardema** [comment]
Yes, I was guilty of that...   Trying to do better.

**Comment by WardF** [comment]
Agreed that we need to get v4.9.0 out, but releasing RC's for public consumption is a best practice.  We will try to shorten the release loop and publish release candidates for 4.9.1 :).

**Comment by seanm** [comment]
4.9.0 shipped, can this be closed?
